spec/rails_helper.rb

```diff
@@ -38,7 +38,7 @@
   # examples within a transaction, remove the following line or assign false
   # instead of true.
   config.use_transactional_fixtures = true
-
+  config.render_views
```

Comment:
including this globally is a bad idea as rendering views is expensive operation and in large projects unnecessarily complicate the data preparation.